test/test-package-tree.ts

```diff
@@ -0,0 +1,30 @@
+import test from 'ava';
+
+import * as tree from '../src/package-tree';
+
+import {TestProject} from './util';
+import {PackageTree} from './util-package-tree';
+
+test('first tester', async t => {
+  const test = new TestProject({
+    '*': ['a@1', 'c@2', 'd@1'],
+    'a@1': ['b@1', 'c@1'],
+    'b@1': ['c@1'],
+    'c@1': [],
+    'c@2': [],
+    'd@1': []
+  });
+  const path: string = await test.create();
+
+  const d1 = new PackageTree<null>('d', '1.0.0', null, []);
```

Comment:
It would be nice to be able to automatically generate this from the argument passed to `TestProject`, _or_ vice-versa (the latter may be better). Just a suggestion for when you populate more tests.